ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template3.kt

```diff
@@ -0,0 +1,199 @@
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.InternalPaywallView
+import com.revenuecat.purchases.ui.revenuecatui.UIConstant
+import com.revenuecat.purchases.ui.revenuecatui.composables.Footer
+import com.revenuecat.purchases.ui.revenuecatui.composables.PaywallIcon
+import com.revenuecat.purchases.ui.revenuecatui.composables.PaywallIconName
+import com.revenuecat.purchases.ui.revenuecatui.composables.PurchaseButton
+import com.revenuecat.purchases.ui.revenuecatui.composables.RemoteImage
+import com.revenuecat.purchases.ui.revenuecatui.data.PaywallViewModel
+import com.revenuecat.purchases.ui.revenuecatui.data.PaywallViewState
+import com.revenuecat.purchases.ui.revenuecatui.data.TestData
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.TemplateConfiguration
+import com.revenuecat.purchases.ui.revenuecatui.data.selectedLocalization
+import com.revenuecat.purchases.ui.revenuecatui.helpers.isInPreviewMode
+
+private object Template3UIConstants {
+    val iconCornerRadius = 8.dp
+    val iconSize = 65.dp
+    val featureIconSize = 35.dp
+    val iconPadding = 5.dp
+}
+
+@Composable
+internal fun Template3(
+    state: PaywallViewState.Loaded,
+    viewModel: PaywallViewModel,
+) {
+    Column(
+        modifier = Modifier.fillMaxSize(),
+    ) {
+        Column(
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxWidth()
+                .padding(
+                    horizontal = UIConstant.defaultHorizontalPadding,
+                    vertical = UIConstant.defaultVerticalSpacing,
+                ),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy(UIConstant.defaultVerticalSpacing, Alignment.CenterVertically),
+        ) {
+            Template3MainContent(state, viewModel)
+        }
+        PurchaseButton(state, viewModel)
+        Footer(templateConfiguration = state.templateConfiguration, viewModel = viewModel)
+    }
+}
+
+@Composable
+private fun Template3MainContent(state: PaywallViewState.Loaded) {
+    IconImage(state.templateConfiguration.images.iconUri)
+    val localizedConfig = state.selectedLocalization
+    val colors = state.templateConfiguration.getCurrentColors()
+    Text(
+        style = MaterialTheme.typography.headlineSmall,
+        fontWeight = FontWeight.Black,
+        textAlign = TextAlign.Center,
+        text = localizedConfig.title,
+        color = colors.text1,
+    )
+    Spacer(modifier = Modifier.height(UIConstant.defaultVerticalSpacing * 2))
+    Features(
+        features = localizedConfig.features,
+        colors = colors,
+    )
+}
+
+@Composable
+private fun Features(
+    features: List<PaywallData.LocalizedConfiguration.Feature>,
+    colors: TemplateConfiguration.Colors,
+) {
+    if (features.isEmpty()) return
+
+    Column(
```

Comment:
I wonder if we should use a `LazyColumn` here... For most cases, this is enough, but if someone ever added a ton of features, `LazyColumn` might make more sense... I think it's ok to keep this though.